src/MediaWiki.ts

```diff
@@ -58,8 +60,10 @@ class MediaWiki {
   public _modulePathOpt: string // only for whiting to generate modulePath
   public webUrl: URL
   public desktopRestApiUrl: URL
+  public mobileRestApiUrl: URL
```

Comment:
Here the same, I see no value of `Rest` in the variable name, to the contrary. Or do I have forgotten something important?

Reply:
Yes, this naming came from url builders, I've updated it where it was possible.